decidim-core/lib/decidim/common_passwords.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Decidim
+  class CommonPasswords
+    include Singleton
+
+    attr_reader :dictionary
+
+    def initialize
+      File.open(File.join(__dir__, "db", "dictionary.txt"), "r") do |file|
```

Comment:
Can we rename the `dictionary.txt` into something more descriptive?

What does this dictionary contain? That should described by its name.